Modules/Core/Common/test/itkOptimizerParametersGTest.cxx

```diff
@@ -52,7 +52,7 @@ TEST(OptimizerParameters, ConstructWithSpecifiedDataAndSize)
 
   // First test for size = zero.
   EXPECT_EQ(OptimizerParametersType(std::vector<double>{}.data(), 0), OptimizerParametersType{});
-  EXPECT_EQ(OptimizerParametersType(std::array<double, 0>{}.data(), 0), OptimizerParametersType{});
+  EXPECT_EQ(OptimizerParametersType(std::array<double, 0>().data(), 0), OptimizerParametersType{});
```

Comment:
Interestingly, GCC4 does not complain about using `{}` for an `std::vector`. 😺 That's because `std::vector<T>` is not a simple aggregate type like `std::array<T, N>`, of course... Anyway, I hope we may soon say goodbye to this old compiler.